pkgs/development/tools/kubectx/default.nix

```
@@ -1,4 +1,4 @@
-{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+{ lib, buildGoModule, fetchFromGitHub, fetchpatch, installShellFiles }:
```

```suggestion
{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
```
fetchpatch is not used?